lib/MetaCPAN/Web/Model/API/Author.pm

```diff
@@ -30,81 +30,27 @@ sub get {
     return $self->request( '/author/' . uc( $author[0] ) )
         if ( @author == 1 );
 
-    return $self->request(
-        '/author/_search',
-        {
-            query => {
-                constant_score => {
-                    filter => { ids => { values => [ map {uc} @author ] } }
-                }
-            },
-            size => scalar @author,
-        }
-    );
-
+    return $self->request( '/author/by_id?id=' . join ',', @author );
```

Comment:
These should be passed as multiple id values, not a comma separated value.

Also, `->request` accepts a hashref of query parameters to add.  If that isn't adequate, we should extend it so this can be passed in in a structured manor.  The other changes in this PR have the same issue.